app/enums.py

```diff
@@ -1,6 +1,13 @@
 from enum import Enum
 
 
+class ProductOFFSource(Enum):
```

Comment:
There is the `Flavor` enum in the Python SDK just for that

Reply:
ok awesome !